SearchQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -55,40 +62,47 @@ Then run the command `docker images`, you will have following images ready:
 2. `opea/web-retriever:latest`
 3. `opea/reranking:latest`
 4. `opea/llm-textgen:latest`
-5. `opea/searchqna:latest`
-6. `opea/searchqna-ui:latest`
+5. `opea/nginx:latest`
+6. `opea/searchqna:latest`
+7. `opea/searchqna-ui:latest`
 
 ## 🚀 Set the environment variables
 
-Before starting the services with `docker compose`, you have to recheck the following environment variables.
+```bash
+cd GenAIExamples/SearchQnA/docker_compose/
+```
+
+Before starting the services with `docker compose`, you have to recheck the following environment variables. Create a file to store
```

Comment:
why here is a linebreak?

Reply:
I found some unneeded line breaks within code blocks and will fix.. Thank you